.circleci/config.yml

```diff
@@ -224,6 +224,20 @@ commands:
           name: Run linter
           command: poetry run poe lint
 
+  lint_rust_runner:
+    description: "Format and lint check of rust test runner"
+    steps:
+      - run:
+          name: Run format check
+          command: cargo fmt --check
+          working_directory: ./protostar-rust
+      - run:
+          name: Run lint check
+          command: cargo clippy --all-targets --all-features -- --no-deps -W clippy::pedantic -A clippy::missing_errors_doc -A clippy::missing_panics_doc -A clippy::default_trait_access
+          working_directory: ./protostar-rust
+          environment:
+            RUSTFLAGS: -Dwarnings
```

Comment:
What does passing it to the compiler do in this case?

Reply:
It makes the check fail even on clippy warnings.